router/src/server.rs

```diff
@@ -570,8 +571,8 @@ async fn chat_completions(
     let stream = req.stream;
     let max_new_tokens = req.max_tokens.or(Some(100));
     let repetition_penalty = req
```

Comment:
@drbh I think this was the wrong parameter.

Reply:
ahh yes, I misunderstood the difference between the frequency and presence and how it mapped to the `repetition_penalty`. Thank you! 🙏